src/quo2/components/new_messages.cljs

```diff
@@ -0,0 +1,74 @@
+(ns quo2.components.new-messages
```

Comment:
`new-messages` is a weird name for a component, where and how it will be used?

Reply:
@flexsurfer - It will be used on the chat screen. Kindly refer to the below Figma design for the usage: https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments?node-id=3773%3A407292

Reply:
oh ok, thank you, probably better then to name it `new-messages-header`

Reply:
Done